apps/src/templates/rubrics/RubricFloatingActionButton.jsx

```diff
@@ -60,6 +60,14 @@ function RubricFloatingActionButton({
     }
   }, [eventData, studentLevelInfo]); // Neither of these should change, so this should run once
 
+  useEffect(() => {
+    const eventName = isOpen
+      ? EVENTS.TA_RUBRIC_OPENED_FROM_FAB_EVENT
+      : EVENTS.TA_RUBRIC_CLOSED_FROM_FAB_EVENT;
+    analyticsReporter.sendEvent(eventName, eventData);
+    trySetSessionStorage(sessionStorageKey, isOpen);
+  }, [isOpen, eventData]);
```

Comment:
Original intent was when the FAB was clicked to indicate the start of using the TA.  I would be interested in having both events if possible - fine to add whichever we don't currently have to next sprint.

Reply:
I rolled back the change I made so that it only fires when the user makes a click, and we can add a ticket to make a different event for if the page loads with it open or something?

Reply:
Sounds great @cearachew ! Can you please make sure that ticket is opened before moving on?

Reply:
Issue created: https://codedotorg.atlassian.net/browse/AITT-493